### PR TITLE
removed tex and man output

### PR DESCRIPTION
**Pull request by JensWehner**
Fix https://github.com/votca/tools/issues/326
blocked by https://github.com/votca/xtp/pull/655
@junghans did I miss anything?

### CODE REVIEW

File: include/votca/tools/property.h [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (junghans) [inline include/votca/tools/property.h:1]: Huh?

**Comment by junghans** [comment]
We will need to merge https://github.com/votca/xtp/pull/655 first as xtp is actually still using this functionality.

**Comment by junghans** [comment]
Wait for https://github.com/votca/votca/pull/593

File: include/votca/tools/propertyiomanipulator.h [hunk 2]
```diff
@@ -29,12 +29,12 @@ namespace tools {
...
  *
  * Changes the state of the output stream. Property class reads this state
- * and formats its output according to this state (XML, TXT, T2T, etc)
+ * and formats its output according to this state (XML, TXT,  etc)
  */
 class PropertyIOManipulator {
 
  public:
-  enum Type { XML, HLP, TEX, TXT };
+  enum Type { XML, HLP, TXT };
```
Comment (junghans) [inline include/votca/tools/propertyiomanipulator.h:38]: Are we using the other types somewhere?

File: include/votca/tools/colors.h [hunk 3]
```diff
@@ -89,45 +89,45 @@ class ColorSchemeBase {
 };
 
 template <typename TColorScheme>
-class Color : public ColorSchemeBase {
+class Color final: public ColorSchemeBase {
```
Comment (junghans) [inline include/votca/tools/colors.h:92]: Are the color schemes used anywhere?

**Review by junghans — CHANGES_REQUESTED** [review]
1 nitpicks and Changleog missing

**Comment by junghans** [comment]
@votca-bot changelog: remove tex and man output from app class

File: include/votca/tools/propertyiomanipulator.h (hunk 2 above)
Comment (JensWehner) [reply include/votca/tools/propertyiomanipulator.h:38]: yes in the property class.

File: include/votca/tools/colors.h (hunk 3 above)
Comment (JensWehner) [reply include/votca/tools/colors.h:92]: yes, they are used in the property class, but I do not think we are using that feature anywhere in particular. So yes and no. 

I think we should remove this step by step.

**Comment by JensWehner** [comment]
@votca-bot format

**Review by junghans — APPROVED** [review]

**Pull request merged**